manage/installing/installation.rst

```diff
@@ -125,11 +125,11 @@ Download the latest Plone unified installer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Download from `the plone.org download page <https://plone.org/download>`_ to your server using wget command. Curl also works.
```

Comment:
How about something simpler like: a portlet that includes links to a list of all releases, in reverse chronological order? The main /download page would still point directly at the latest release. That way, no extra code, and no human maintenance required.